### ISSUE TITLE
[SF][Table]: table header row's role is not accurate

### ISSUE THREAD

**Issue by summer2200** [SAP/ui5-webcomponents#10574]
### Describe the bug

An a11y issue has been reported which says the table's header is not announced as header row when focused.
Currently it has `role="row"` but seems not accurate.
![Image](https://github.com/user-attachments/assets/99ea8c83-aadd-4e3b-be32-4b89db499f6f)

### Reproduction steps

1.
2.
3.
...

### Expected Behaviour

When table header row is focused, it should be identified as header row and read out with screen reader on.

### Screenshots or Videos

https://github.com/user-attachments/assets/4495f28c-798f-47db-82de-35d084478ac9

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by IlianaB** [comment]
Hello @summer2200 ,
Is this issue about the same table, you are reporting in issue https://github.com/SAP/ui5-webcomponents/issues/10471 ?

**Comment by summer2200** [comment]
Hi @IlianaB , these two issues are not the same. This one is about table header's role, and https://github.com/SAP/ui5-webcomponents/issues/10471 is to let table row being read out with each column's header. Please check the video attached in https://github.com/SAP/ui5-webcomponents/issues/10471.

**Comment by IlianaB** [comment]
Hello @summer2200 ,
I understand there is a difference between both issues reported. I just want to know which table is this issue about, so that I can forward to the responsible component, developing the exact table.

**Comment by summer2200** [comment]
Hi @IlianaB , yes these two issues are about the same table.

**Comment by summer2200** [comment]
Hi @IlianaB , yes these two issues are about the same table.

**Comment by nnaydenow** [comment]
Hello @SAP/ui5-webcomponents-table,

Could you please check the issue? The author confirm that the table is the same to the one mentioned in related issue (not compatibility table)

Regards,
Nayden

**Comment by JumpNRun** [comment]
I've created an internal incident for your issue: DINC0391950. We will update you as soon as possible.
